index.js

```diff
@@ -138,13 +138,26 @@ var Node = function (path, opts, disposable) {
     daemonPid: function () {
       return this.subprocess && this.subprocess.pid
     },
-    getConf: function (key, cb) {
+    getConfig: function (key, cb) {
+      if (typeof key === 'function') {
+        cb = key
+        key = ''
+      }
       var t = this
       var result = ''
       run(IPFS_EXEC, ['config', key], {env: t.env})
         .on('error', cb)
         .on('data', function (data) { result += data })
-        .on('end', function () { cb(null, result.trim()) })
+        .on('end', function () {
+          cb(null, result.trim())
+        })
+    },
+    setConfig: function (key, value, cb) {
+      var t = this
```

Comment:
I'm interested to learn what you would recommend for us to adopt from ES6 throughout our Node.js stuff. 

Why would var self = this be an anti-pattern? I know that now with fat arrows, we can avoid having to do that, but that can be more of a style preference than actually an anti-pattern, right?